### PR TITLE
[1LP][RFR] Azone Graph Initial commit

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================

- Added `wait_candu_data_available` method in Azone Model Page
- Added `Utilization` navigation in the model page with returning View
- Added `AzoneCloudUtilizationView` in candu view page
- Added Hourly graph support test in new `cfme/candu/azone.py` file.

{{pytest: cfme/tests/candu/test_azone_graph.py -v --use-provider complete}}

### CODE REVIEW

File: cfme/cloud/availability_zone.py [hunk 1]
```diff
@@ -162,3 +177,12 @@ class AvailabilityZoneTimelines(CFMENavigateStep):
 
     def step(self, *args, **kwargs):
         self.prerequisite_view.toolbar.monitoring.item_select('Timelines')
+
+
+@navigator.register(AvailabilityZone, "candu")
```
[the remark this answers is not in the document]
Comment (digitronik) [reply cfme/cloud/availability_zone.py]: @mshriver agreed, I added changes for `Azone`, I will fallow this for upcoming `Cluster` and `Datastore`. I will raise one PR for already pushed like `VM` and `Host`... :+1:

File: cfme/tests/candu/test_azone_graph.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+@pytest.fixture(scope="function")
+def azone(appliance, provider):
+    collection = appliance.collections.cloud_av_zones
+    azone_name = provider.data["cap_and_util"]["capandu_azone"]
+    return collection.instantiate(name=azone_name, provider=provider)
+
+
+def compare_data(table_data, graph_data, legends, tolerance=1):
+    """ Compare Utilization graph and table data.
+    Args:
```
Comment (rlbabyuk) [inline cfme/tests/candu/test_azone_graph.py]: Required: Space is needed after title

File: cfme/tests/candu/test_azone_graph.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+                                leg=leg, dt=row["Date/Time"]
+                            )
+                        )
+
+
+@pytest.mark.parametrize("interval", INTERVAL)
+@pytest.mark.parametrize("graph_type", GRAPHS)
+def test_graph_screen(provider, azone, graph_type, interval, enable_candu):
+    """Test Availibility zone graphs for Hourly
+    prerequisites:
```
Comment (rlbabyuk) [inline cfme/tests/candu/test_azone_graph.py]: Required: Space here

File: cfme/tests/candu/test_azone_graph.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+def compare_data(table_data, graph_data, legends, tolerance=1):
+    """ Compare Utilization graph and table data.
+
+    Args:
+        table_data : Data from Utilization table
+        graph_data : Data from Utilization graph
+        legends : Legends in graph; which will help for comparison
+        tolerance : Its error which we have to allow while comparison
+    """
+    for row in table_data:
```
Comment (psav) [inline cfme/tests/candu/test_azone_graph.py]: **REQUIRED:** Here it is again ;)

**Comment by digitronik** [comment]
for `compare_data` method need to merge: https://github.com/ManageIQ/integration_tests/pull/7473

**Comment by digitronik** [comment]
@psav Changes added...

**Pull request merged**